autodp/cdf_bank.py

```diff
@@ -0,0 +1,93 @@
+"""
+The work flow of AFA: https://arxiv.org/abs/2106.08567)
+Step 1. composes the log of characteristic functions.
+Step 2. To convert back to delta(eps) or eps(delta), we will use numerical inversion to
+convert the characteristic function back to CDFs.
+This module provides two numerical inversion approaches to implement the second step.
+cdf_quad: Implement Levy theorem using Gaussian quadratures (recommend).
+cdf_fft: Numerical inversion through FFT (see https://www.tandfonline.com/doi/abs/10.1080/03461238.1975.10405087).
+"""
+import numpy as np
+import time
+import scipy.integrate as integrate
+from scipy.fft import fft
+from scipy.stats import norm, laplace
+
+
+def cdf_gaussian(params, x):
+    """
+    The cdf of a Gaussian distribution evaluated at the position x
+
+    params: params['sigma'] denotes the noisy scale.
+    x: evaluates the cdf at x.
+    """
+    sigma = params['sigma']
+    mean = params['mean']
```

Comment:
The mean is not used?  Also this function and the cdf_laplace function seem be to utility functions in that they do not correspond to dominating pairs of any functions?   I thought cdf_bank contains functions to instantiate mechanisms described by their dominating pairs using cdf_p and and cdf_q?

Reply:
Maybe I shall remove the cdf_bank.py as we do not use it?